src/layouts/Layout.astro

```diff
@@ -101,7 +101,7 @@ const {
   </head>
   <body>
     <Header />
-    <main class="grow py-14">
+    <main class="py-14 w-full max-w-[calc(1130px+8vw)] mx-auto">
```

Comment:
1. `grow` appears to be a flex class, but without a flex container, it doesn't make sense. So, I've removed it.
2. I got `max-w-[calc(1130px+8vw)]` from the navigation bar and container max width. I don't like the fact that we are duplicating it, but we might have to change it in multiple places to make it controlled from a single place.